service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/AliasServiceImpl.java

```diff
@@ -270,7 +270,7 @@ public int updateKCUsersGuids() {
 						Collectors.mapping(row -> {
 							UUID userGuid = row.getUUID(0);
 							UUID pushId = row.getUUID(2);
-							String pushName = pushApplicationService.getPushName(pushId);
+							String pushName = pushApplicationService.getClientName(pushId);
```

Comment:
there in not reason creating a method called `getPushName` and then rename to a `getClientName` on a different commit on the same branch. **simply rewrite your branch history so that I'll always be called the name you wish it to be called**